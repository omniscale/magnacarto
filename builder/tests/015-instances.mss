#foo {
    [type='foo'] {
        line-width: 1;
    }
    a/polygon-fill: black;
    b/building-height: 0.5;
    a/line-width: 10;
    b/line-width: 5;
    b/line-color: blue;
    a/line-color: red;

    [type='bar'] {
        line-width: 2;
    }
}