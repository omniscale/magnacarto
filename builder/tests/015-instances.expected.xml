<Map srs="epsg:3857">
  <Parameters></Parameters>
  <Style name="foo" filter-mode="first">
    <Rule>
      <Filter>([type] = &#39;bar&#39;)</Filter>
      <PolygonSymbolizer fill="#000000"></PolygonSymbolizer>
      <LineSymbolizer stroke="#ff0000" stroke-width="10"></LineSymbolizer>
      <LineSymbolizer stroke="#0000ff" stroke-width="5"></LineSymbolizer>
      <LineSymbolizer stroke-width="2"></LineSymbolizer>
    </Rule>
    <Rule>
      <Filter>([type] = &#39;foo&#39;)</Filter>
      <LineSymbolizer stroke-width="1"></LineSymbolizer>
      <PolygonSymbolizer fill="#000000"></PolygonSymbolizer>
      <LineSymbolizer stroke="#ff0000" stroke-width="10"></LineSymbolizer>
      <LineSymbolizer stroke="#0000ff" stroke-width="5"></LineSymbolizer>
    </Rule>
    <Rule>
      <PolygonSymbolizer fill="#000000"></PolygonSymbolizer>
      <LineSymbolizer stroke="#ff0000" stroke-width="10"></LineSymbolizer>
      <LineSymbolizer stroke="#0000ff" stroke-width="5"></LineSymbolizer>
    </Rule>
  </Style>
  <Layer name="foo" srs="" status="off">
    <StyleName>foo</StyleName>
  </Layer>
</Map>